desktop/core/requirements.txt

```diff
@@ -62,6 +61,7 @@ slack-sdk==3.2.0
 SQLAlchemy==1.3.8
 sqlparse==0.4.1
 tablib==0.13.0
+tabulate==0.8.9
```

Comment:
To make it work with Py2, you would need to unzip the pypi package into https://github.com/cloudera/hue/tree/master/desktop/core/ext-py and `git add` it